apps/web/styles/app.css

```diff
@@ -19,7 +19,7 @@ body,
 #main {
   position: relative;
   display: grid;
-  grid-template-rows: 56px 1fr 100px;
+  grid-template-rows: 56px 56px 1fr 100px;
```

Comment:
This will have to be conditional in the future.

Reply:
Addressed in https://github.com/status-im/status-web/pull/362.